src/utils/parse-args.ts

```diff
@@ -0,0 +1,12 @@
+export function parseArgs(args: string = ''): Record<string, string> {
```

Comment:
```suggestion
// NOTE(Xe): This code is temporary and was added on 2025-11-14.
export function parseArgs(args: string = ''): Record<string, string> {
```

I have added a comment noting when this was added so we can use shame to ensure it is in fact temporary :)

Reply:
Removed this file